modules/config/src/main/scala/scala/cli/config/Keys.scala

```diff
@@ -341,7 +342,7 @@ object Keys {
 
   val publishCredentials: Key[List[PublishCredentials]] = new Key[List[PublishCredentials]] {
     override val description: String =
-      "Publishing credentials, syntax: s1.oss.sonatype.org value:user value:password"
```

Comment:
it's nice to have an example in a guide or general command doc, but IMO reference docs & help outputs (like here) should provide general syntax instead
tl;dr don't keep it here, but it may be worth having in a guide doc or smth

Reply:
There are examples of the publish credentials already in `publish` and `publish setup` docs.